carbon-reductor-core/src/main/java/de/envite/greenbpm/carbonreductor/core/adapter/carbonawarecomputing/CarbonAwareComputingMapper.java

```diff
@@ -11,7 +11,7 @@ public class CarbonAwareComputingMapper {
     public EmissionTimeframe mapToDomain(EmissionsData emissionsData) {
         return new EmissionTimeframe(
                 new OptimalTime(emissionsData.getTimestamp()),
-                new EarliestForecastedValue(0.0),
```

Comment:
@maxbehr801 WattTime works like a charm, see next Screenshot
<img width="1296" alt="C8_WattTime" src="https://github.com/user-attachments/assets/6f8f8e85-ce9e-4da0-a50e-0873efd5f713">

But I think the Carbon Aware Computing API is not mapped correctly, could you please verify my assumption and corresponding improvement here:
If no shifting is needed all values are 0 which should not be correct, see next image:
<img width="1301" alt="C8_Computing_Api_2" src="https://github.com/user-attachments/assets/6e9cc8bc-ac90-4e14-9585-14d64d507882">

Because the API returned a value for the optimal execution time which is likely the current timeframe:
<img width="1296" alt="C8_Computing_Api" src="https://github.com/user-attachments/assets/03a052f2-49ab-41c3-92e3-a91ad42ce8ba">